source/run-command/tests/unified/runCommand.json

```diff
@@ -108,7 +108,13 @@
               "commandStartedEvent": {
                 "command": {
                   "ping": 1,
-                  "$db": "db"
+                  "$db": "db",
```

Comment:
should we update the name of the test case here to reflect the other assertions for the lsid and readpreference? also I recall we discussed adding tests for attaching an explicit session and the non-default readpreference, or are those already here?

Reply:
- [Explicit session test: "attaches the provided session lsid to given command"](https://github.com/mongodb/specifications/blob/DRIVERS-2577-runCommand/source/run-command/tests/unified/runCommand.yml#L70)
- [Non-default RP test: "attaches the provided $readPreference to given command"](https://github.com/mongodb/specifications/blob/DRIVERS-2577-runCommand/source/run-command/tests/unified/runCommand.yml#L89)